README.md

```diff
@@ -12,6 +12,20 @@ Javascript, Lua, Ocaml, Pascal, Perl, PHP, Prolog, Python, Ruby, Rust, Scheme.
 A markdown description of the input in English and French can also be
 generated.
 
+## Installation
+
+You can install `iorgen` in a virtual environment like this:
+
+```
+git clone git@github.com:prologin/iorgen.git
+cd iorgen
+python3 -m venv .venv
+. .venv/bin/active
```

Comment:
Use `source` rather than `.`, there's better shell support for it and it's more explicit/intelligible.